client/app/components/Story/StoryActions.jsx

```diff
@@ -10,6 +10,7 @@ import {
   faDoorOpen,
   faDoorClosed,
   faExclamationTriangle,
+  faEyeSlash,
```

Comment:
You'll want to update this [part](https://github.com/ifmeorg/ifme/blob/main/client/app/components/Story/StoryActions.jsx#L76) of the file to use this new icon.